src/commands/ext-update.ts

```diff
@@ -54,6 +54,15 @@ export default new Command("ext:update <instanceId>")
       const sourceUrl = await resolveSource(currentSpec.name);
       const newSource = await extensionsApi.getSource(sourceUrl);
       const newSpec = newSource.spec;
+      if (currentSpec.version === newSpec.version) {
+        utils.logLabeledBullet(
+          logPrefix,
+          `${clc.bold(instanceId)} is already up to date. Its version is ${clc.bold(
```

Comment:
I'm wondering how we want to talk about "Its version".  To me, this current text says "the version of your extension is X.X.X". But is it _actually_ that "the version of the official Firebase extension is X.X.X"?  And that your extension instance uses vX.X.X of the extension?

Do we want to describe an individual extension instance as having a version?  It's subtle, but I'm asking so that we can be consistent here and in the docs :-)

Reply:
Well, both have the same version in this scenario. The version of your extension instance is the same as the latest version of the official extension. (And yes extension instances have versions, just like how your computer may run a specific version of Node. The Node "installation" on your computer has a version, which may be the same or different than the latest version of Node)

Reply:
Cool.  This makes logical sense and SGTM!

Reply:
you ok with this logic, @bkendall ?

Reply:
Fine by me :)